framework/src/modules/auth/constants.ts

```diff
@@ -22,3 +22,7 @@ export const MESSAGE_TAG_TRANSACTION = Buffer.from('LSK_TX_', 'utf8');
 export const ADDRESS_LENGTH = 20;
 export const ED25519_PUBLIC_KEY_LENGTH = 32;
 export const ED25519_SIGNATURE_LENGTH = 64;
+
+export const defaultConfig = {
```

Comment:
Not sure, but just as a comment: Don't you need to update the auth/types.ts file here?

Reply:
I consider creating a new type just to create an object that contains 1 `number` property to be an overkill. I assessed the correct type directly in the module:

```typescript
private _moduleConfig!: { maxNumberOfSignatures: number };
```

```typescript
validator.validate<{ maxNumberOfSignatures: number }>(configSchema, config);
```

If more configurable properties are added to the module, then we can consider creating a dedicated type for it.

Reply:
Ok sounds good, resolving the comment.